.travis.yml

```diff
@@ -25,10 +25,8 @@ services:
 matrix:
   include:
     - env: DOCKER_VERSION=indigo-desktop-trusty
-    - env: DOCKER_VERSION=jade-desktop-trusty
     - env: DOCKER_VERSION=kinetic-desktop-xenial
     - env: DOCKER_VERSION=lunar-desktop-xenial
-    - env: DOCKER_VERSION=lunar-desktop-stretch
```

Comment:
Unrelated change ?

Reply:
Since it's not displayed in the README, there is no reason to keep it in the Travis Matrix

Reply:
Stretch has proven to be a valuable CI platform as many system packages are different from Xenial (because Stretch is much more recent). I'd rather keep it. Feel free to add it to the README if that's the only motivation for removing it.